src/dynarec/la64/dynarec_la64_helper.h

```diff
@@ -433,6 +433,188 @@
         BSTRINS_D(wback, ed, wb2 + 7, wb2); \
     }
 
+#define YMM0(a)                                             \
+    do {                                                    \
+        if (dyn->lsx.avxcache[a].v != -1) {                 \
+            dyn->lsx.avxcache[a].zero_upper = 1;            \
+            dyn->lsx.avxcache[a].width = LSX_AVX_WIDTH_128; \
```

Comment:
Is this line necessary?

Reply:
I guess not.
Maybe assert here is better, or just use zero_upper=1 is enough.